pkg/services/plugindashboards/plugindashboards.go

```diff
@@ -17,21 +18,24 @@ type pluginSettingsStore interface {
 }
 
 func ProvideService(sqlStore *sqlstore.SQLStore, bus bus.Bus, pluginStore plugins.Store,
-	pluginDashboardManager plugins.PluginDashboardManager, dashboardImportService dashboardimport.Service) *Service {
-	s := new(sqlStore, bus, pluginStore, pluginDashboardManager, dashboardImportService)
+	pluginDashboardManager plugins.PluginDashboardManager, dashboardImportService dashboardimport.Service,
+	pluginSettingsStore *pluginsettings.Service) *Service {
+	s := new(sqlStore, bus, pluginStore, pluginDashboardManager, dashboardImportService, pluginSettingsStore)
 	s.updateAppDashboards()
 	return s
 }
 
 func new(pluginSettingsStore pluginSettingsStore, bus bus.Bus, pluginStore plugins.Store,
```

Comment:
Next step would be to consolidate `pluginSettingsStore` and `pluginsettings.Store` probably, right @marefr?

Reply:
I was wondering about the same thing

Reply:
Yeah good point. `pluginSettingsStore` was introduced to be able to mock the store. I think `pluginSettingsStore` should be replaced with `pluginsettings.Service`.

Reply:
should that happen on this PR?

Reply:
Up to you